utilities/automake.mk

```diff
@@ -1,9 +1,12 @@
 bin_PROGRAMS += \
 	utilities/ovs-appctl \
-	utilities/ovs-testcontroller \
 	utilities/ovs-dpctl \
 	utilities/ovs-ofctl \
 	utilities/ovs-vsctl
+if !P4OVS
```

Comment:
Blank line before the `if !P4OVS`? You've done this elsewhere in the file.

Reply:
done